src/Libraries/Nop.Services/Customers/Caching/CustomerAddressMappingCacheEventConsumer.cs

```diff
@@ -10,13 +10,19 @@ namespace Nop.Services.Customers.Caching
     public partial class CustomerAddressMappingCacheEventConsumer : CacheEventConsumer<CustomerAddressMapping>
     {
         /// <summary>
-        /// Clear cache data
+        /// Clear cache by entity event type
         /// </summary>
         /// <param name="entity">Entity</param>
+        /// <param name="entityEventType">Entity event type</param>
         /// <returns>A task that represents the asynchronous operation</returns>
-        protected override async Task ClearCacheAsync(CustomerAddressMapping entity)
+        protected override async Task ClearCacheAsync(CustomerAddressMapping entity, EntityEventType entityEventType)
         {
-            await RemoveByPrefixAsync(NopCustomerServicesDefaults.CustomerAddressesPrefix);
+            await RemoveAsync(NopCustomerServicesDefaults.CustomerAddressesCacheKey, entity.CustomerId.ToString());
```

Comment:
I see no reason to use ToString

Reply:
This is a small optimization so we avoid extra boxing (int -> object).

Reply:
Instead, convert to a string, I'm not even sure which is faster, in any case, such an entry looks redundant

Reply:
Returned as it was, in a single case it will not give any advantages